packages/react/src/generators/remote/schema.json

```diff
@@ -28,6 +28,12 @@
       "type": "string",
       "enum": ["as-provided", "derived"]
     },
+    "dynamic": {
+      "type": "boolean",
+      "description": "Should the host application use dynamic federation?",
+      "default": false,
+      "x-priority": "internal"
```

Comment:
This _could_ be left open if the user generates a new remote to an existing host.